clients/cobol-lsp-vscode-extension/src/extension.ts

```diff
@@ -60,3 +61,26 @@ export function activate(context: ExtensionContext) {
     context.subscriptions.push(disposable);
     item.text = "Language Server is active";
 }
+
+async function isJavaInstalled() {
+    return new Promise<any>((resolve, reject) => {
+        const ls = cp.spawn("java", ["-version"]);
+        ls.stderr.on("data", (data: any) => {
+            if (!data.toString().includes('java version "1.8')) {
+                reject("Java version 8 expected");
+            }
+            resolve();
+        });
+        ls.on("error", (code: any) => {
+            if ("Error: spawn java ENOENT" === code.toString()) {
+                reject("Java 8 is not found");
+            }
+            reject(code);
+        });
+        ls.on("close", (code: number) => {
+            if (code !== 0) {
+                reject("An error occured when checking if Java was installed");
```

Comment:
pls fix the typo.. I checked the overall behaviour and it works fine!

Reply:
Typo fixed, change has been pushed in the repository